src/components/Table/GridTable.tsx

```diff
@@ -327,37 +337,48 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
         );
       // Even if we don't pass the filter, one of our children might, so we continue on after this check
       let isCard = false;
+
       if (matches) {
-        isCard = nestedCards && nestedCards.maybeOpenCard(row);
+        isCard = rowCards && rowCards.maybeOpenCard(row);
         filteredRows.push([row, makeRowComponent(row)]);
       }
 
       const isCollapsed = collapsedIds.includes(row.id);
       if (!isCollapsed && !!row.children?.length) {
-        nestedCards && matches && nestedCards.addSpacer();
+        rowCards && matches && rowCards.addSpacer();
         visitRows(row.children, isCard);
       }
 
-      isCard && nestedCards && nestedCards.closeCard();
+      isCard && rowCards && rowCards.closeCard();
     }
 
     function visitRows(rows: GridDataRow<R>[], addSpacer: boolean): void {
       const length = rows.length;
       rows.forEach((row, i) => {
         if (row.kind === "header") {
-          nestedCards && nestedCards.maybeOpenCard(row);
+          // Flag to determine if the header has nested card styles.
```

Comment:
Just curious, does schedules v2 header have a nested card style? 

...I kinda wonder if @bdow 's maybe forthcoming/WIP refactoring of "for leaf-rows, use a real border" refactoring would clean this up, b/c we could assume that headers only ever have one row, hence are defacto leaf rows.

That musing aside, we should keep this approach for now/this PR

Reply:
SchedulesV2 header doesn't need to be nested cards but due to the left and right padding columns, we need it to spread the background colours, alignment and other things that we didn't fully figure out.

It would for sure benefit from @bdow refactor so don't need to call these open cards and deal with CSS selectors with interesting counts to compensate for open and close rows.